Data/DataModel/Event.cs

```diff
@@ -233,6 +233,12 @@ public bool AreStandingsAvailableNow
         /// </summary>
         public bool HideHints { get; set; }
 
+        /// <summary>
+        /// The number of hint coins a team must have to unlock help threads for any puzzle.
+        /// Note that this can be overridden by the puzzle itself as well.
+        /// </summary>
+        public uint DefaultCostForHelpThread { get; set; }

```

Comment:
It doesn't *really* matter, but I am wondering why this is a uint whereas the puzzle property is an int. They actually get stored in the DB as differently-sized fields (and uint is apparently larger FWIW). If you're fine with things as they are, I'm happy to go with it because the event table is exceedingly small, but I will hold merging until you ring in.

Reply:
Thanks for the catch! I updated event to be int to match Hint and Puzzle